arcgis-android-toolkit/build.gradle

```diff
@@ -75,18 +75,19 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlin_coroutines"
-    implementation "com.android.support:appcompat-v7:$droid.support"
-    implementation "com.android.support.constraint:constraint-layout:$droid.constraintLayout"
-    implementation "com.android.support:multidex:$droid.multiDex"
-    implementation "android.arch.lifecycle:runtime:$droid.lifecycle"
-    api "android.arch.lifecycle:common-java8:$droid.lifecycle"
+    implementation "androidx.appcompat:appcompat:$droid.appcompat"
+    implementation "androidx.constraintlayout:constraintlayout:$droid.constraintLayout"
+    implementation "androidx.multidex:multidex:$droid.multiDex"
+    implementation "androidx.lifecycle:lifecycle-runtime:$droid.lifecycle"
+    api "androidx.lifecycle:lifecycle-common-java8:$droid.lifecycle"
```

Comment:
Why is this a transitive dependency while the others aren't? @LDuncAndroid - do you remember if there was a specific reason?

Reply:
Because we don't need to expose the rest of the libs, we just need an interface from `androidx.lifecycle:lifecycle-common-java8:$droid.lifecycle`

Reply:
Oh I see, you are saying because we expose types of `androidx.lifecycle` in the public API of the toolkit, it needs to be a transitive dependency, for example `ArcGISArView` implements the `DefaultLifecycleObserver`.

Reply:
It's just for the public method on `ArcGISArView` `registerLifecycle(LifecycleOwner)`.